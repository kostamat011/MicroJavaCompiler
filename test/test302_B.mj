//Test 302

program test302

const int jedan = 1;

int i;
bool ibool;

int niz2;

int niz[];
int a;

{
	void printArgs(int num, char... args)
		int i;
	{
		i = 0;
		do{
			print(args[i]);	
			i++;
		}while(i < num);
	}
	
	void main()
	{
		print(eol);
		printArgs(5,'t','e','s','t','B');
	}
}

