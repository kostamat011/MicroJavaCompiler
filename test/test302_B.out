    0    1v   97
v    4
   35
   40   30   60  120
18900
testB
Completion took 1 ms